test/es-module/test-esm-exports.mjs

```diff
@@ -70,6 +70,7 @@ import fromInside from '../fixtures/node_modules/pkgexports/lib/hole.js';
     ['pkgexports/null', './null'],
     ['pkgexports/invalid2', './invalid2'],
     ['pkgexports/invalid3', './invalid3'],
+    ['pkgexports/invalid5', 'invalid5'],
```

Comment:
Is it worth double-checking the error message here? I'm never sure which bits apply to `require` vs. `import`.

Reply:
AFAICT we don't have to worry about require vs. import here since we are handling the case in the error handler itself rather than in the loader. If we wanted to do a more specific check on the message we would have to further refactor this test.

Reply:
I'm not sure if it passes through the JS error mapping for the ESM resolver which is written on the C++ side: https://github.com/nodejs/node/blob/0fac393d263fc7e2f4f054c9d4aab0c1c3cf00c8/src/module_wrap.cc#L879-L917

Reply:
At the very least the reference to "C++" is outdated now. Both resolvers are now implemented in JS although they're still separate.